package.json

```diff
@@ -2,18 +2,23 @@
   "name": "socket.io-parser",
   "version": "2.2.6",
   "description": "socket.io protocol parser",
+  "files": [
+    "index.js",
+    "binary.js",
+    "is-buffer.js"
+  ],
   "repository": {
     "type": "git",
     "url": "https://github.com/Automattic/socket.io-parser.git"
   },
   "dependencies": {
-    "debug": "2.2.0",
-    "json3": "3.3.2",
-    "component-emitter": "1.1.2",
-    "isarray": "0.0.1",
-    "benchmark": "1.0.0"
+    "component-emitter": "^1.2.0",
+    "debug": "^2.2.0",
+    "isarray": "^1.0.0",
+    "json3": "^3.3.2"
```

Comment:
I don't know where the original issue is, but I think that's because ambiguous dependencies can introduces unexpected bugs which we can't control.